omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
+
+# Global FIPS override flag.
+if windows? || rhel?
+  override :fips, enabled: true
+end
 
 dependency "preparation"
-dependency "chef"
-dependency "pry"
+dependency "rb-readline" if windows?
```

Comment:
rb-readline is purely ruby.  It was made to avoid native gems.  I've verified by installing it without a compiler on my path on windows (unless some sneaky magic was happening).

That said...  *shrug*  who knows.  We can try turning it on everywhere.  If it works, it would save us some effort trying to build libedit for ruby.